src/pybind/mgr/dashboard_v2/tools.py

```diff
@@ -106,10 +107,6 @@ class BaseController(six.with_metaclass(BaseControllerMeta, object)):
     """
     _mgr_module = None
 
-    _cp_config_default = {
```

Comment:
Why are you removing this from the `BaseControllerMeta`?

Reply:
`AuthRequired()` can now be used independent of `BaseController`. `ApiRoot` is not a controller